app/konnect/deployment/import.md

```diff
@@ -0,0 +1,136 @@
+    organization, we recommend always setting this flag to avoid accidentally
+    pushing configuration to the wrong group.
+
+1. Log in to your [{{site.konnect_saas}}](http://cloud.konghq.com/login) account.
+
+1. From the left navigation menu, open **Runtimes**, then open the runtime group
+you just updated.
+
+1. Look through the configuration details of any imported entities to make sure
+they were migrated successfully.
+
+## Migrate data planes
+
+You can keep any data plane nodes that are:
+* Running {{site.base_gateway}} (not the open-source package)
+* Are at least version 2.3 or higher
+
+Turn any self-managed nodes into cloud data plane nodes by registering them
+through the Runtime Manager and adjusting their configurations, or power down
+the old instances and create new data plane nodes through {{site.konnect_saas}}.
+
+1. Follow the [runtime setup guide](/konnect/runtime-manager/#kong-gateway) for
+your preferred deployment type.
+
+2. Once you have created or converted the data plane nodes, `kong stop` your
+old Gateway runtimes, then shut them down.
+
+3. If any of the old nodes have connected PostgreSQL or Cassandra instances,
+you can shut them down now.
+
+## Post-migration tasks
+
+See the following docs to set up any additional things you may need:
+
+* **Dev Portal files:** You can migrate API specs and markdown documents using the
+{{site.konnect_saas}} GUI. See [Dev Portal Service Documentation](/konnect/dev-portal/service-documentation).
+
+* **Dev Portal applications and developers:** If you have developers or
+applications registered through the Portal, those developers need to create new
+accounts in {{site.konnect_saas}} and register their applications in the new
+location.
+    * [Developer registration](/konnect/dev-portal/access-and-approval/dev-reg)
+    * [Enable application registration](/konnect/dev-portal/applications/enable-app-reg):
+    App registration in {{site.konnect_saas}} works through a different
+    mechanism than in self-managed {{site.base_gateway}}. Enable app
+    registration on each service that requires it.
+    * [Publish Services to the Dev Portal](/konnect/dev-portal/publish):
+    The Dev Portal is automatically enabled on a {{site.konnect_saas}} org
+    (Plus or Enterprise tier). Publish your services to the Dev Portal.
+* [**Prepare custom plugins for migration**](/konnect/configure/servicehub/manage-plugins/#custom-plugins):
+Custom plugins are supported in {{site.konnect_saas}}, but with limitations. As
+long as your plugins fit the criteria, or if you can adjust them to do so,
+contact Kong Support to get the plugin manually added to your account.
+* [**Review and set up teams and roles**](/konnect/org-management/teams-and-roles):
+{{site.konnect_saas}} groups and roles don't map directly to
+{{site.base_gateway}} teams and roles. Set up teams to mirror your
+{{site.base_gateway}} groups, then invite users to your {{iste.konnect_saas}}
+org and assign them to a team on invite.
```

Comment:
Thought: In the near future, this may simply require an IdP migration. 

If IdP is setup, then the users and team mappings will be provisioned by the IdP. Let's revisit this once we have an update with Generic OIDC providers. (Near-term post-helium feature)